### DISCUSSION TITLE
Grafana Dashboard

### DISCUSSION THREAD

**Discussion by JamesonBeebe** [srsran/srsRAN_4G#1320] [Q&A] [answered]
I see there is a Grafana dashboard GUI available for the srsRAN project, but it doesn't look like srsRAN 4G is supported, or at least not documented. 
[https://docs.srsran.com/projects/project/en/latest/user_manuals/source/grafana_gui.html#launching-gui](url)

Is there support to get the Grafana dashboard working with srsRAN 4G or has anyone had success with this?

**Comment by brendan-mcauliffe** [comment, marked as answer]
Hi @JamesonBeebe srsRAN 4G does not have a Grafana dashboard. You can get some visual information using srsGUI though, you can find more info [here](https://github.com/srsran/srsGUI).
